Fails to install playwright dependency

Why do it even need playwright in production mode?
Do we really need it?


` "postinstall": "npx playwright install",`



```
FROM node:16-slim
 RUN npm install --omit=dev -g react-use-measure@2.1.6
```

```
npm ERR! code 127
npm ERR! path /usr/local/lib/node_modules/react-use-measure
npm ERR! command failed
npm ERR! command sh -c -- npx playwright install
npm ERR! npm WARN exec The following package was not found and will be installed: playwright@1.50.0
npm ERR! npm WARN EBADENGINE Unsupported engine {
npm ERR! npm WARN EBADENGINE   package: 'playwright@1.50.0',
npm ERR! npm WARN EBADENGINE   required: { node: '>=18' },
npm ERR! npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm ERR! npm WARN EBADENGINE }
npm ERR! npm WARN EBADENGINE Unsupported engine {
npm ERR! npm WARN EBADENGINE   package: 'playwright-core@1.50.0',
npm ERR! npm WARN EBADENGINE   required: { node: '>=18' },
npm ERR! npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm ERR! npm WARN EBADENGINE }
npm ERR! sh: 1: playwright: not found
```

--- a/package.json
+++ b/package.json
@@ -58,9 +58,8 @@
     }
   },
   "scripts": {
-    "postinstall": "npx playwright install",
     "dev": "vite",
     "build": "rimraf dist && vite build && tsc",
-    "test": "vitest run"
+    "test": "npx playwright install && vitest run"
   }
 }

